resources/js/views/Teams/CreateTeam.vue

```diff
@@ -0,0 +1,188 @@
+<template>
+    <div class="ctc">
+        <h1 class="title is-2">Create a Team</h1>
+
+        <p class="mb2">You are allowed to create {{ this.remaining }} team(s)</p>
+
+        <div class="columns mt3">
+
+            <div class="column is-one-third">
+                <p class="mb1">What kind of Team would you like to create?</p>
```

Comment:
Should define translate for this.

Reply:
There are a lot of missing translations. Right now I am still working on the functionality